src/components/CartItemCard/CartItemCard.tsx

```diff
@@ -0,0 +1,71 @@
+import Button from '../common/Button/Button';
+import Divider from '../common/Divider/Divider';
+import QuantityStepper from '../common/QuantityStepper/QuantityStepper';
+import { CartItem } from '../../types/type';
+import useDeleteCartItem from '../../hooks/useDeleteCartItem';
+import usePatchCartItem from '../../hooks/usePatchCartItem';
+
+import * as S from './CartItemCard.style';
+
+interface CartItemProps {
+  cartItem: CartItem;
+}
+
+function CartItemCard({ cartItem }: CartItemProps) {
+  const { quantity } = cartItem;
+
+  // TODO: ProductItem과 공통 로직 별도 hook으로 분리해볼까
+  const deleteCartItemMustation = useDeleteCartItem();
+  const patchCartItemMutation = usePatchCartItem();
+
+  const handleDeleteCartItem = () => {
+    deleteCartItemMustation.mutate({ cartItemId: cartItem.id });
+  };
+
+  const handleIncreaseQuantity = () => {
+    patchCartItemMutation.mutate({
+      cartItemId: cartItem.id,
+      quantity: quantity + 1,
+    });
+  };
+
+  const handleDecreaseQuantity = () => {
+    if (quantity === 1) {
+      deleteCartItemMustation.mutate({ cartItemId: cartItem.id });
+    } else {
+      patchCartItemMutation.mutate({
+        cartItemId: cartItem.id,
+        quantity: quantity - 1,
+      });
+    }
```

Comment:
```suggestion
    if (quantity === 1) {
      deleteCartItemMustation.mutate({ cartItemId: cartItem.id });
      return
    }
    patchCartItemMutation.mutate({
      cartItemId: cartItem.id,
      quantity: quantity - 1,
    });
```
이렇게 적으면 depth가 조금 줄어들 것 같은데 어떻게 생각하시나요?

Reply:
복잡한 코드가 아니고, if 및 else의 뒤에 따라오는 코드가 delete와 patch로 엄연히 다르다 보니,`else`를 사용해서 quantity가 1일때와 아닐때에 대해서 논리 구조를 명확하게 하고자 하였습니다.
하지만 블링이 얘기해 주신 대로 else를 사용하지 않고, early return 을 사용해서 가독성과 유지보수성을 챙기는 것이 더 좋아보입니다 :)

else문을 사용한 곳들을 early return으로 변경해 주었습니다.

e7a5d23